Snakemake/config/tsinfer_cattle.yaml

```diff
@@ -1,45 +0,0 @@
-PROJECT: "africa"
-workdir: "/exports/cmvm/eddie/eb/groups/HighlanderLab/gfortuna/giro_popgen/tree_seq_pipeline/Snakemake/{PROJECT}"
-species: "Bos taurus"
-ploidy: "2"
-noChromosomes: 29
-vcfDir: "/exports/cmvm/eddie/eb/groups/HighlanderLab/gfortuna/giro_popgen/tree_sequence/cattle"
-genmap: "/exports/cmvm/eddie/eb/groups/HighlanderLab/gfortuna/giro_popgen/resources/ma2015_shapeit_ARS-UCD1.2"
-#combinedVCFs: null
-merge: "yes"
-ancestralAllele: "/exports/cmvm/eddie/eb/groups/HighlanderLab/gfortuna/giro_popgen/resources/ancestral_alleles/cattle_ancestral.txt"
-meta: "/exports/cmvm/eddie/eb/groups/HighlanderLab/gfortuna/giro_popgen/tree_sequence/cattle/files/metadata_afr.txt"
-chromosome_length:
-  1 : 158534110
-  2 : 136231102
-  3 : 121005158
-  4 : 120000601
-  5 : 120089316
-  6 : 117806340
-  7 : 110682743
-  8 : 113319770
-  9 : 105454467
-  10 : 103308737
-  11 : 106982474
-  12 : 87216183
-  13 : 83472345
-  14 : 82403003
-  15 : 85007780
-  16 : 81013979
-  17 : 73167244
-  18 : 65820629
-  19 : 63449741
-  20 : 71974595
-  21 : 69862954
-  22 : 60773035
-  23 : 52498615
-  24 : 62317253
-  25 : 42350435
-  26 : 51992305
-  27 : 45612108
-  28 : 45940150
-  29 : 51098607
-  X : 139009144
-  Y : 43300181
-  M : 16340
-
```

Comment:
Why is this file gone?

Reply:
because this was the one for my run. Don't need to track here (we have a general one already from the tutorial)